src/app/applications/applications.controller.app.js

```diff
@@ -15,6 +16,23 @@
     $scope.findAvatar = rUtils.findRandomAvatar;
     $scope.findIcon = rUtils.findIcon;
     $scope.uploadIcon = uploadIcon;
+    $scope.selectFile = selectFile;
+    $scope.onFile = onFile;
+    $scope.cropImage = cropImage;
+    $scope.cropper = {};
+    $scope.cropperProxy = 'cropper.first';
+    $scope.showEvent = 'show';
+    $scope.hideEvent = 'hide';
+    $scope.options = {
+        maximize: true,
+        aspectRatio: 2 / 1,
+        crop: function(dataNew) {
+          data = dataNew;
+        }
+    };
+    $scope.filename = '';
+    $scope.showButton = false;
+    var file, data;
```

Comment:
There should not be such local variables.
Either they are "global" within the scope, or they are local to the functions.